main

```diff
@@ -0,0 +1,124 @@
+import asdasd
+import telebot
+import requests
+from peewee import *
+from asdasd import *
+import datfile
+parameters = {
+    #'q': 'dollar', # query phrase  # maximum is 100
+    'country': 'gb',
+    #'source': None,
+    'category': 'business',
+    'apiKey': '0b19c9bde21541859e8176ed2060de79' # your own API key
+}
+import json
+from telebot import types
+#import TicTacToe
+url = 'https://newsapi.org/v2/top-headlines?'
+parameters = {
+    #'q': 'dollar', # query phrase  # maximum is 100
+    'country': 'gb',
+    #'source': None,
+    'category': 'business',
+    'apiKey': '0b19c9bde21541859e8176ed2060de79' # your own API key
+}
+db = SqliteDatabase('db/database.db')
+class sus():
+    id = PrimaryKeyField(unique=True)
+    country = CharField()
+    category = CharField()
+    ever = IntegerField()
+
+bot = telebot.TeleBot('5247888467:AAH2hPIdtmH26P3mhdCKkpSua9Pt44mYoY0');
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv